frontend/src/Frontend/UI/Dialogs/AccountDetails.hs

```diff
@@ -35,14 +37,15 @@ uiAccountDetails
   :: ( HasUiAccountDetailsModelCfg mConf key t
      , MonadWidget t m
      )
-  => IntMap.Key
-  -> Account key
+  => NetworkName
+  -> AccountName
```

Comment:
Can we obtain this by calling `accountName` on the `Account` argument?

Reply:
Yep, also the account details dialog is being upgraded to work on both vanity & non-vanity accounts, so this will likely change with my PR.